site/content/en/docs/Installation/Creating Cluster/gke.md

```diff
@@ -66,6 +66,55 @@ To install `gcloud` and `kubectl`, perform the following steps:
 
 [gcloud-install]: https://cloud.google.com/sdk/docs/quickstarts
 
+### Choosing a GKE cluster mode
+
+A [cluster][cluster] consists of at least one *control plane* machine and multiple worker machines called *nodes*. In Google Kubernetes Engine, nodes are [Compute Engine virtual machine][vms] instances that run the Kubernetes processes necessary to make them part of the cluster.
+
+Agones supports GKE Standard mode and has alpha support for GKE Autopilot mode.
+Agones `GameServer` and `Fleet` manifests that work on Standard are compatible
+on Autopilot with some constraints, described in the following section. You
+can't convert existing Standard clusters to Autopilot; create new Autopilot
+clusters instead.
+
+#### Agones on GKE Autopilot
+
+Autopilot is GKE's fully-managed mode. GKE configures, maintains, scales, and
+upgrades nodes for you, which can reduce your maintenance and operating
+overhead. You only pay for the resources requested by your running Pods, and
+you don't pay for unused node capacity or Kubernetes system workloads.
+
+This section describes the Agones-specific considerations in Autopilot
+clusters. For a general comparison between Autopilot and Standard, refer to
+[Choose a GKE mode of operation](https://cloud.google.com/kubernetes-engine/docs/concepts/choose-cluster-mode).
+
+Autopilot nodes are, by default, optimized for most workloads. If some of your
+workloads have broad compute requirements such as Arm architecture or a minimum
+CPU platform, you can also choose a
+[compute class](https://cloud.google.com/kubernetes-engine/docs/concepts/autopilot-compute-classes)
+that meets that requirement. However, if you have specialized hardware needs
+that require fine-grained control over machine configuration, consider using
+GKE Standard.
+
+Agones on Autopilot has pre-configured opinionated constraints. Evaluate
+whether these constraints impact your workloads:
+
+*  **Operating system:** No Windows containers.
+*  **Resource requests:** Autopilot has pre-determined
+   [minimum Pod resource requests](https://cloud.google.com/kubernetes-engine/docs/concepts/autopilot-resource-requests#min-max-requests).
+   If your game servers require less than those minimums, use GKE Standard.
+*  **[Scheduling strategy](https://pkg.go.dev/agones.dev/agones/pkg/apis#SchedulingStrategy):** `Packed` is supported, which is the Agones default.
+   `Distributed` is not supported.
+*  **[Host port policy]({{<ref "/docs/reference/agones_crd_api_reference#agones.dev/v1.GameServerPort">}}):** `Dynamic` is supported, which is the Agones default.
+   `Static` and `Passthrough` are not supported.
+*  **Seccomp profile:** Agones sets the seccomp profile to `Unconfined` to
+   avoid unexpected container creation delays that might occur because
+   Autopilot enables the `RuntimeDefault` seccomp profile.
+*  **[Pod disruption policy]({{<ref "/docs/Advanced/controlling-disruption#eviction-api">}}):**
```

Comment:
I always thought that `ref` shortcode needed the ".md" at the end, but seems like they don't. TIL!

https://gohugo.io/functions/ref/